code/modules/clothing/head/mind_monkey_helmet.dm

```diff
@@ -55,9 +60,40 @@
 		UnregisterSignal(magnification, COMSIG_SPECIES_LOSS)
 		magnification = null
 		visible_message(span_notice("[src] falls silent and drops on the floor. Maybe you should try again later?"))
-		playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
+		var/particle_path
+		switch(rage_chance)
+			if(-7 to 0)
+				user.visible_message(span_notice("[src] falls silent and drops on the floor. Try again later?"))
+				playsound(src, 'sound/machines/buzz-sigh.ogg', 30, TRUE)
+				particle_path = null
+			if(7 to 13)
```

Comment:
all looks good just change these numbers to defines :+1:

Reply:
sure klet me just make it more confusing by turning the incrementing the easily readable 7% probability into random arbitrary defines that cant be understood intuitively at a glance

Reply:
make them easy to understand at a glance then